### PR TITLE
become_user inside block does not inherit become applied to block, triggers 501

### PR DESCRIPTION
**Pull request by m-a-leclercq**
Shows #705

### CODE REVIEW

File: test/role-block/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
```
Comment (webknjaz) [inline test/role-block/tasks/main.yml]: This needs to be integrated into tests. Meaning there should be a test that runs this and then checks that there's no errors of a specific type.

Comment (m-a-leclercq) [reply test/role-block/tasks/main.yml]: I'm a bit lost as to what needs to be done, I've read `contributing.md` but that doesn't really tell me much as to how I'm supposed to create my test and push my code. Any documentation I migh have missed that would help me? 
Thanks

Comment (webknjaz) [reply test/role-block/tasks/main.yml:1]: Sorry, I missed your comment. @ssbarnea has created a reproducer test @ #793. Also, I've added a few references on adding pytest based tests @ https://github.com/ansible/ansible-lint/issues/725.

Comment (ssbarnea) [reply test/role-block/tasks/main.yml:1]: @webknjaz You can close this one now.

**Pull request closed without merging**